ada_feeding/README.md

```diff
@@ -57,6 +60,13 @@ Note the following design paradigms when writing this subclass of `ActionServerB
 - Only this file should need to import the ROS action type for the action server. The specific behaviors in the tree should not need access to that information.
 - ROS action preemption requests are implemented using the `stop()` pathway of the behavior tree, which internally calls `terminate()` on each behavior. Hence, `terminate()` must properly implement everything that needs to be done when the ROS action preempts. (It should already be implemented this way for good behavior tree design.)
 
+### Subclassing `MoveToTree`
+
+All robot motion actions publish the same feedback and result (see [`ada_feeding_msgs`](https://github.com/personalrobotics/ada_feeding/tree/amaln/goal_constraints/ada_feeding_msgs/action)). Hence, the shared logic of publishing that feedback and result is implemented in the `MoveToTree`. Any tree that publishes that feedback and result should subclass `MoveToTree` and  implement the abstract function `create_move_to_tree`.
+
+### Creating Robot Motions with Arbitrary Goal and Path Constraints
+Robot motion is implemented as a behavior, `MoveTo`. By itself, the behavior will do nothing, since no goal or path constraints will be set. Goal and path constraints are implemented as decorators on top of the `MoveTo` behavior, that can be chained together in order to have multiple constraints. See the already implemented trees (e.g., `MoveToConfiguration`, `MoveToConfigurationWithPosePathConstraints`) for examples.
+
```

Comment:
I ran this with a slightly older version of rclpy and was getting errors when stopping execution in sim. Can we add a note in this doc that if you're getting the following error in the action server, to make sure rclpy is updated? 

Error: the following exception was never retrieved: cannot use destroyable because destruction was requested